circleci/images/failtester/files/sbin/install-testdeps

```diff
@@ -20,23 +25,42 @@ apt-get install -y --no-install-recommends \
   apt-transport-https \
   ca-certificates \
   curl \
-  debian-archive-keyring
+  debian-archive-keyring \
+  gnupg
+
+# import key and keep apt-key from issuing warnings
+echo "Importing PGDG key..." >&2
+curl -sf https://www.postgresql.org/media/keys/ACCC4CF8.asc | \
+APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=1 apt-key add -
+
+# add pgdg repo to sources
+echo "Writing /etc/apt/sources.list.d/pgdg.list..." >&2
+echo "deb http://apt.postgresql.org/pub/repos/apt/ ${codename}-pgdg main 12" > \
+     /etc/apt/sources.list.d/pgdg.list  
 
-pkg_cloud_url="https://packagecloud.io/install/repositories/citusdata/the-process/script.deb.sh"
+pkg_cloud_url="https://packagecloud.io/install/repositories/citus-bot/the-process/script.deb.sh"
```

Comment:
why is this url changed? Same for below

Reply:
I dont have access to citusdata packagecloud, I only have access to citus-bot so I have uploaded the packages to citus-bot.